### PR TITLE
Update error format

### PR DESCRIPTION
**Pull request by liubo0127**
@LinuxGit @superlzs0476 PTAL

### CODE REVIEW

File: callback_plugins/help.py [hunk 1]
```diff
@@ -48,29 +47,67 @@ def __init__(self):
...
+        dumped = '\n  '.join(dumped.split('\n')).rstrip()
+        return dumpd_tile + dumped + '\n'
 
     def print_help_message(self):
-        self._display.display("Ask for help:", color=C.COLOR_WARN)
-        self._display.display("Contact us: support@pingcap.com", color=C.COLOR_HIGHLIGHT)
+        self._display.display("Ask TiDB User Group for help:", color=C.COLOR_WARN)
         self._display.display(
-            "It seems that you encounter some problems. You can send an email to the above email address, attached with the tidb-ansible/inventory.ini and tidb-ansible/log/ansible.log files and the error message, or new issue on https://github.com/pingcap/tidb-ansible/issues. We'll try our best to help you deploy a TiDB cluster. Thanks. :-)",
+            "It seems that you have encountered some problems, you can describe the operation steps and error information as much as possible on https://asktug.com. We will do our best to help you solve the problem.. Thanks. :-)",
```
Comment (liubo0127) [inline callback_plugins/help.py]: @lilin90  PTAL

Comment (lilin90) [inline callback_plugins/help.py]:
```suggestion
            "It seems that you have encountered some problem. Please describe your operation steps and provide error information as much as possible on https://asktug.com (in Chinese) or https://stackoverflow.com/questions/tagged/tidb (in English). We will do our best to help solve your problem. Thanks. :-)",
```

Comment (lilin90) [reply callback_plugins/help.py]: FYI. `some` can mean `a certain` when you don't know what thing exactly. @liubo0127

File: callback_plugins/help.py [hunk 2]
```diff
@@ -48,29 +47,67 @@ def __init__(self):
...
+
+        # remove invocation unless specifically wanting it
+        if not keep_invocation and self._display.verbosity < 3 and 'invocation' in abridged_result:
+            del abridged_result['invocation']
+
+        # remove diff information from screen output
+        if self._display.verbosity < 3 and 'diff' in abridged_result:
+            del abridged_result['diff']
+
+        if 'access_control_allow_headers' in abridged_result:
```
Comment (qrr1995) [inline callback_plugins/help.py]: `dict.pop()` is more elegant , see https://stackoverflow.com/questions/11277432/how-to-remove-a-key-from-a-python-dictionary

**Pull request merged**